.travis.yml

```diff
@@ -7,14 +7,6 @@ branches:
     - /^\d\.\d+\.\d+(rc\d+|\.dev\d+)?$/
 matrix:
   include:
-    - env: TOXENV=py27
-      python: 2.7
-    - env: TOXENV=py27-pinned
-      python: 2.7
-    - env: TOXENV=py27-extra-deps
-      python: 2.7
-    - env: TOXENV=pypy
-      python: 2.7
```

Comment:
The deploy line still needs feedback @Gallaecio @kmike

Reply:
If you mean line 63 below, I think we just need to change `$TOXENV == py27` to `$TOXENV == py37`.

Reply:
done